spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -223,80 +247,6 @@
     end
   end
 
-  describe "#assign_visibility?" do
-    context "when no params are specified" do
-      it "does not need to assign visibility" do
-        expect(actor.send(:assign_visibility?)).to eq false
-      end
-    end
-
-    context "when file set params with visibility are specified with symbols as keys" do
-      let(:file_set_params) { { visibility: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC } }
-
-      it "does need to assign visibility" do
-        expect(actor.send(:assign_visibility?, file_set_params)).to eq true
-      end
-    end
-
-    context "when file set params with visibility are specified with strings as keys" do
-      let(:file_set_params) { { "visibility" => Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC } }
-
-      it "does need to assign visibility" do
-        expect(actor.send(:assign_visibility?, file_set_params)).to eq true
-      end
-    end
-  end
-
-  describe "#set_representative" do
```

Comment:
It was in integrations.  But for ease of mind over what is, effectively, two lines of code and to get away from overuse of integration testing, I added some explicit tests in the actor spec for `#attach_to_work`:
```ruby
    context 'without representative and thumbnail' do
      it 'assigns them (with persistence)' do
        actor.attach_to_work(work)
        expect(work.representative).to eq(file_set)
        expect(work.thumbnail).to eq(file_set)
        expect { work.reload }.not_to change { [work.representative, work.thumbnail] }
      end
    end

    context 'with representative and thumbnail' do
      it 'does not (re)assign them' do
        allow(work).to receive(:thumbnail_id).and_return('ab123c78h')
        allow(work).to receive(:representative_id).and_return('zz365c99q')
        expect(work).not_to receive(:representative=)
        expect(work).not_to receive(:thumbnail=)
        actor.attach_to_work(work)
      end
    end
```

I trust this is the last we'll have to talk about how the logic is.